### PR TITLE
RZ: Inline Formula

### PR DESCRIPTION
**Pull request by ax3l**
## Description

GitHub now supports inline LaTeX in Markdown, so we can remove our manual image for the RZ mode definition.

Also slightly clarify the wording on the ordering of modes - mode 0 is special (real only).

## Affected Components

- `base`

## Logic Changes

None.

## Writer Changes

None affected.

## Reader Changes

None affected.

## Data Converter

None affected.

### CODE REVIEW

File: STANDARD.md [hunk 1]
```diff
@@ -366,7 +366,9 @@ meshes):
       - for `thetaMode` geometry:
         - `m=3;imag=+` (3 *modes* and using a `+` sign for the definition of
                         the *imaginary* part)
-                       ![definition of imaginary part](img/cylindrical.png)
+```math
+    E_z = \tilde{E}_{z,0} + \sum_{l=1}^{m} \mathcal{R}(\tilde{E}_{z,l}) \cdot \cos(l \Theta) {\color{red}+} \mathcal{I}(\tilde{E}_{z,l}) \cdot \sin(l \Theta)$
```
Comment (RemiLehe) [inline STANDARD.md]:
```suggestion
    E_z = \mathcal{R}(\tilde{E}_{z,0}) + \sum_{l=1}^{m} \mathcal{R}(\tilde{E}_{z,l}) \cdot \cos(l \Theta) {\color{red}+} \mathcal{I}(\tilde{E}_{z,l}) \cdot \sin(l \Theta)$
```

Comment (RemiLehe) [reply STANDARD.md]: Just to make this extra clear?

**Review by RemiLehe — CHANGES_REQUESTED** [review]

File: STANDARD.md (hunk 1 above)
Comment (ax3l) [reply STANDARD.md]: Does it make it clearer though? I thought $E_{z,0}$ is purely rational - could this maybe confuse people?

Comment (ax3l) [reply STANDARD.md]: Hm, nah I think it's fine.

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**